### PR TITLE
chore(clickhouse): bump up zookeeper dependency chart

### PR DESCRIPTION
**Pull request by prashant-shahi**
This PR fixes the issue seen in K8s version > 1.21, when replica count of Zookeeper is set to more than 1.

Issue:

```
Error: UPGRADE FAILED: resource mapping not found for name: "my-release-zookeeper" namespace: "platform" from "": no matches for kind "PodDisruptionBudget" in version "policy/v1beta1"
ensure CRDs are installed first
```
Signed-off-by: Prashant Shahi <prashant@signoz.io>

### CODE REVIEW

File: charts/clickhouse/Chart.yaml [hunk 1]
```diff
@@ -2,16 +2,17 @@ apiVersion: v2
...
 appVersion: "22.8.8"
 icon: https://raw.githubusercontent.com/ClickHouse/ClickHouse/master/website/images/logo-400x240.png
 sources:
   - https://github.com/signoz/charts/
   - https://github.com/ClickHouse/ClickHouse
 dependencies:
   - name: zookeeper
-    repository: https://raw.githubusercontent.com/bitnami/charts/pre-2022/bitnami
-    version: 7.0.5
+    repository: oci://registry-1.docker.io/bitnamicharts
```
Comment (srikanthccv) [inline charts/clickhouse/Chart.yaml:13]: How does this `oci::` work?

Comment (prashant-shahi) [reply charts/clickhouse/Chart.yaml:13]: OCI (Open Container Initiative) is a project that defines open standards for container formats and runtime.
Big companies and cloud platforms are their members.

Back in January, Bitnami introduced support for OCI and they plan to eventually moving out of _index.yaml_ chart release.
https://blog.bitnami.com/2023/01/bitnami-helm-charts-available-as-oci.html

For the end-user, there would not be any significant change.
You can find the OCI-based charts repository in DockerHub: https://hub.docker.com/r/bitnamicharts/zookeeper

Comment (srikanthccv) [reply charts/clickhouse/Chart.yaml:13]: Do all helm versions support oci scheme?

Comment (prashant-shahi) [reply charts/clickhouse/Chart.yaml:13]: @srikanthccv It supports starting from Helm v3.8.0.

**Review by srikanthccv — APPROVED** [review]
Add a note about the help version. I think it didn't work for me for old version.

**Comment by prashant-shahi** [comment]
@srikanthccv did you get to test it with newer Helm version?

**Comment by srikanthccv** [comment]
Yes, I updated, and it seems to be working now.

```
helm version
version.BuildInfo{Version:"v3.10.0", GitCommit:"ce66412a723e4d89555dc67217607c6579ffcb21", GitTreeState:"clean", GoVersion:"go1.19.1"}
```

**Pull request merged**